indico/modules/events/editing/service.py

```diff
@@ -148,3 +154,48 @@ def service_handle_new_editable(editable):
     except requests.RequestException as exc:
         logger.exception('Failed calling listener for editable')
         raise ServiceRequestFailed(exc)
+
+
+def service_handle_review_editable(editable, parent_revision, revision=None):
+
+    class ReviewEditableSchema(mm.Schema):
+        publish = fields.Boolean(default=True)
+        comment = fields.String()
+        comments = fields.List(fields.String())
+        tags = fields.List(fields.String())
+
+    # TODO: consider not sending the endpoints
+    new_revision = revision or parent_revision
+    data = {
+        'revision': EditingRevisionUnclaimedSchema().dump(new_revision),
+        'endpoints': {
```

Comment:
Passing the endpoints as part of the request starts to get excessive when multiple service endpoints might need them all together. I'm starting to considering these should be extraneous to the type of request but, instead, be built by the micro-service. IMO, we should document these endpoints and let the service build the ones it intends to use.

Reply:
We are no longer considering this.